docs/deploy-scl3.rst

```diff
@@ -0,0 +1,208 @@
+* Check that the Kuma_ and the KumaScript_ builds are passing.
+
+.. _Kuma: https://travis-ci.org/mozilla/kuma/
+.. _KumaScript: https://travis-ci.org/mdn/kumascript
+
+* Create a pre-push branch from master::
+
+    git remote -v | grep origin
+    # origin  git@github.com:mozilla/kuma.git (fetch)
+    # origin  git@github.com:mozilla/kuma.git (push)
+    git fetch origin
+    git checkout -b pre-push-2017-08-08 origin/master
+
+* Update the submodules::
+
+    git submodule update --init
+    cd kumascript
+    git fetch origin
+    git checkout origin/master
+    cd ..
+    cd locale
+    git fetch origin
+    git checkout origin/master
+    cd ..
+    git diff
+    git commit kumascript locale
+
+  For bonus points, the commit message should mention the KumaScript PRs::
+
+    Update locales, KumaScript PRs 235,271,275,277,278,280
+
+    * mdn/kumascript#235 - TopicBox: Add de translation
+    * mdn/kumascript#271 - GroupData: Update WebVR sidebar
+    * mdn/kumascript#275 - Move mozilla/kumascript to mdn/kumascript
+    * mdn/kumascript#277 - EmbedInteractiveExample: Update URL
+    * mdn/kumascript#278 - GroupData: Add Nav.mediaDevices.getUserMedia
+    * mdn/kumascript#280 - Spec2: Mark SIMD Obsolete
+
+  This will add references to the PRs back to the commit where there were
+  deployed. You can see the commits from the `What's Deployed`_ page, or by
+  using `GitHub's compare view`_.
+
+.. _`What's Deployed`: https://whatsdeployed.io/s-FHK
+.. _`GitHub's compare view`: https://github.com/blog/612-introducing-github-compare-view
+
+* Push the branch and open a pull request::
+
+    git push -u origin pre-push-2017-08-08
+
+* Check that tests pass. The ``TOXENV=locales`` job, which uses Dennis_ to
+  check strings, is the job that occasionally fails. This is due to a
+  string change committed in Pontoon_ that will break rendering in production.
+  If this happens, fix the incorrect translation, push to master on
+  mdn-l10n_, and update the submodule in the pull request. Wait for the
+  tests to pass.
+
+.. _Dennis: https://github.com/willkg/dennis
+.. _Pontoon: https://pontoon.mozilla.org/projects/mdn/
+.. _mdn-l10n: https://github.com/mozilla-l10n/mdn-l10n
+
+* Merge the PR. A "Rebase and merge" is preferred, to avoid a merge commit
+  with little long-term value. Delete the pre-push branch.
+
+
+Deploy to Staging
+-----------------
+The staging site is located at https://developer.allizom.org.  It runs on the
+same Kuma code as production, but against a different database, other backing
+services, and with less resources. It is used for verifying code changes before
+pushing to production.
+
+* Start the staging push. There are two methods for pushing, the chief website
+  (a.k.a the Big Red Button) and the command line tool james_. Both require VPN
+  access, and a commit hash you want to push.  ``james`` defaults to your current working
+  copy::
+
+    git fetch origin
+    git checkout origin/master
+    james stage
```

Comment:
I think, this need to be run in docker or virtualenv if `chief-james` is not installed in the local machine. Or you can specify that `pip install chief-james`